src/templates/en/2020/stories/user_experience.html

```diff
@@ -0,0 +1,2526 @@
+            </div>
+          </div>
+          <div data-amp-original-style="position:absolute;pointer-events:none;left:12.13592%;top:75.08091%;width:65.53398%;height:11.8123%;opacity:1" class="amp-wp-a82bca4">
+            <div id="el-be776817-2be7-4595-a6a4-8dbe2e96cb6d" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0" class="amp-wp-a1b70b8">
+              <p class="fill amp-wp-2d1cef5" data-amp-original-style="white-space:pre-wrap;overflow-wrap:break-word;word-break:break-word;margin:-1.0000000000000004% 0;font-family:'Lato',sans-serif;font-size:0.291262em;line-height:1.5;text-align:initial;padding:0% 0%;color:#000000"><span data-amp-original-style="color: #5b6770" class="amp-wp-66da2af">The results show that almost half of the websites in the dataset have included a privacy policy.</span>
+            </p></div>
+          </div>
+          <div data-amp-original-style="position:absolute;pointer-events:none;left:11.40777%;top:59.70874%;width:47.08738%;height:18.12298%;opacity:1" class="amp-wp-68bd11b">
+            <div id="el-bcdafe65-4b2c-4f7e-97e7-a54a7d280952" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0" class="amp-wp-a1b70b8">
+              <p class="fill amp-wp-1f4abec" data-amp-original-style="white-space:pre-wrap;overflow-wrap:break-word;word-break:break-word;margin:6.185567010309279% 0;font-family:'Poppins',sans-serif;font-size:1.294498em;line-height:1.1;text-align:initial;padding:0% 0%;color:#000000"><span data-amp-original-style="font-weight: 700; color: #5c687d" class="amp-wp-3f0135d">45</span><span
+                data-amp-original-style="font-weight: 800; color: #5c687d" class="amp-wp-b6cd492">%</span></p></div>
+          </div>
+        </div>
+      </div>
+    </amp-story-grid-layer>
+    <amp-story-page-attachment layout="nodisplay" href="https://almanac.httparchive.org/en/2020/privacy" data-cta-text="More facts about Privacy" class="i-amphtml-layout-nodisplay" hidden="hidden" i-amphtml-layout="nodisplay"></amp-story-page-attachment>
+  </amp-story-page>
+  <amp-story-page id="7526112f-7bd8-4e4c-8c0e-2eef65e77b8d" auto-advance-after="15s" class="i-amphtml-layout-container" i-amphtml-layout="container">
+    <amp-story-grid-layer template="vertical" aspect-ratio="412:618" class="grid-layer i-amphtml-layout-container" i-amphtml-layout="container">
+      <div class="page-fullbleed-area amp-wp-c84718f" data-amp-original-style="background-color:#fff">
+        <div class="page-safe-area">
+          <div data-amp-original-style="position:absolute;pointer-events:none;left:0%;top:-9.25926%;width:100%;height:118.51852%;opacity:1" class="amp-wp-0e01dfe">
+            <div class="mask amp-wp-a1b70b8" id="el-d5fef557-db69-4f81-ad2a-a00085c1a25b" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0">
```

Comment:
This id (`el-d5fef557-db69-4f81-ad2a-a00085c1a25b`) is duplicated

Reply:
Argh, I really need to find out what's causing [this bug](https://github.com/google/web-stories-wp/issues/5137). Good catch!